pkgs/by-name/sv/svchttpx/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, fetchFromGitHub, graalvm17-ce, maven, fetchurl }:
+let
+  mavenJdk17 = maven.override {
+    jdk = graalvm17-ce;
+  };
+  metadataVersion = "0.3.4";
+in
+mavenJdk17.buildMavenPackage rec {
+  pname = "svchttpx";
```

Comment:
Not sure about the name, but `httpx` is [already taken](https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/tools/security/httpx/default.nix#L27)...

Reply:
httpx-cli`?

Reply:
thanks, good idea :+1: